client/components/ui/files/RenameDialogFooter.vue

```diff
@@ -0,0 +1,22 @@
+<script lang="ts" setup>
+import type { DynamicDialogInstance } from 'primevue/dynamicdialogoptions'
+import type { ComputedRef } from 'vue/dist/vue'
+
+const { t } = useI18n()
+const dialog = inject<ComputedRef<DynamicDialogInstance>>('dialogRef')
```

Comment:
this applies to all provide/injects (so ctrl-shift-f the project) : use a Symbol (better security, and it carries the type of the injected value, removing the need for the generic on call site)

Reply:
can dialog really be undefined ? Is there a situation where no dialog is provided ? If not, add a bang at the end of the inject, and remove the `?.` throughout the file

Reply:
I cannot use symbol in this case, since the `provide()` side is handled by PrimeVue. I will look for a symbol, but quite sure there isn't one.

Regarding dialog being undefined, well if this happens, I guess PrimeVue must be broken somehow.

Reply:
cringe primevue